common/views/components/Image/Image.js

```diff
@@ -78,7 +78,9 @@ const Img = ({
       height={height}
       className={classNames({
         image: true,
-        'bg-charcoal font-white': true,
+        'bg-charcoal font-white': Boolean(
+          extraClasses && !extraClasses.match(/bg-/)
```

Comment:
This couples class naming pretty tightly here.

Why doesn't something like `bg-blue` just override this class?
And do we then assume that the new background should have black text?

If there are only a couple of combos we could use an enum like
```js
import { Image, ImageBackgrounds } from './Image'
//...

<Image classNames={classNames({ ImageBackground.dark: true })} />
```
```

Reply:
Ping @davidpmccormick